web/src/topic_list.ts

```diff
@@ -337,7 +337,7 @@ export function initialize({
 }): void {
     $("#stream_filters").on(
         "click",
-        ".sidebar-topic-check, .topic-name, .topic-markers-and-controls",
+        ".sidebar-topic-check, .sidebar-topic-name, .topic-markers-and-controls",
```

Comment:
This line in `topic_popover.js` looks like it might need to also be updated, with some care around how the popover is also declared for `.inbox-row .inbox-topic-menu`

```
                    url = $elt.closest("li").find(".topic-name").expectOne().prop("href");              
```

Reply:
Great catch. The `if (instance.reference.classList.contains("inbox-topic-menu"))` conditional appears to isolate the line you pointed out safely from the Inbox popover, and manual testing confirms that. But maybe there's something else I'm overlooking where I'm not exercising due care?

Reply:
Ahh, yes, quite so, reading the code to notice that block is the "some care" required :)